e2eAIOK/deltatuner/deltatuner/deltatuner_model.py

```diff
@@ -262,6 +255,45 @@ def from_pretrained(
         model.load_adapter(model_id, adapter_name, is_trainable=is_trainable, **kwargs)
         return model
 
+    def _get_denas_config(self, model_id: str, **kwargs: Any):
```

Comment:
The same question as before, why we need to initiate the deltatuner here, which will limit the user to change the deltatuner arguments. If we want to provide the APIs for downloading the necessary configurations from the HF website, I think we can override some arguments with the downloaded arguments.

Reply:
Same as above response, not repeat it here.